### PR TITLE
www-apps/redmine: bump 3.4.x branch to 3.4.2, added ruby23 support

### PR DESCRIPTION
**Pull request by winterheart**
Package-Manager: Portage-2.3.6, Repoman-2.3.1

### CODE REVIEW

**Comment by winterheart** [comment]
@SoapGentoo done.

**Comment by winterheart** [comment]
@mgorny done.

File: www-apps/redmine/redmine-3.3.4.ebuild [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
-# ruby23 don't have required deps
+# ruby24 don't have required deps
```
Comment (mgorny) [inline www-apps/redmine/redmine-3.3.4.ebuild]: This is irrelevant change in the first commit. Use `git rebase -i` to edit it and amend it back to ruby23. Then update it in third.

File: www-apps/redmine/redmine-3.4.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,221 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+# ruby24 dion't have required deps
```
Comment (mgorny) [inline www-apps/redmine/redmine-3.4.2.ebuild]: I'd say either leave it at 23 and fix afterwards, or at least fix the typo here.

File: www-apps/redmine/redmine-3.2.7-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,215 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (mgorny) [inline www-apps/redmine/redmine-3.2.7-r1.ebuild]: Please say what you're fixing in the summary line. Bug numbers are not very useful for someone looking through `git log`. You can mention it in footer using `Bug: https://bugs.gentoo.org/NNNNNN`.

**Pull request closed without merging**